### PR TITLE
Populating email and phone fields in citizen apis

### PR DESCRIPTION
**Pull request by DommarajuUmamaheswari**

### CODE REVIEW

File: Solution/src/Kmd.Momentum.Mea/Citizen/Model/CitizenData.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace Kmd.Momentum.Mea.Citizen.Model
+{
+    public class CitizenData
```
Comment (amitxagarwal) [inline Solution/src/Kmd.Momentum.Mea/Citizen/Model/CitizenData.cs:5]: why this new class? can we add the contact information in the existing citizendataresponsemodel?

File: Solution/src/Kmd.Momentum.Mea/Citizen/Model/ContactInformation.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Kmd.Momentum.Mea.Citizen.Model
+{
+    public class ContactInformation
+    {
+        public Email Email { get; set; }
```
Comment (amitxagarwal) [inline Solution/src/Kmd.Momentum.Mea/Citizen/Model/ContactInformation.cs:5]: how can we be sure if email is just gonna be single email and not an array or list of email id's?

File: Solution/src/Kmd.Momentum.Mea/Citizen/Model/ContactInformation.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+        public Phone Phone { get; set; }
```
Comment (amitxagarwal) [inline Solution/src/Kmd.Momentum.Mea/Citizen/Model/ContactInformation.cs:6]: same comment for phone no.

File: Solution/src/Kmd.Momentum.Mea/Citizen/Model/CitizenData.cs (hunk 1 above)
Comment (DommarajuUmamaheswari) [reply Solution/src/Kmd.Momentum.Mea/Citizen/Model/CitizenData.cs:5]: we have to map this object to response class so we have added another class

File: Solution/src/Kmd.Momentum.Mea/Citizen/Model/ContactInformation.cs (hunk 2 above)
Comment (DommarajuUmamaheswari) [reply Solution/src/Kmd.Momentum.Mea/Citizen/Model/ContactInformation.cs:5]: Got clarification from momemtum team on this and email is not an array.

File: Solution/src/Kmd.Momentum.Mea/Citizen/Model/ContactInformation.cs (hunk 3 above)
Comment (DommarajuUmamaheswari) [reply Solution/src/Kmd.Momentum.Mea/Citizen/Model/ContactInformation.cs:6]: Got clarification from momemtum team on this and phone is not an array.

**Pull request merged**